test/models/character_test.rb

```diff
@@ -55,4 +55,24 @@ class CharacterTest < ActiveSupport::TestCase
     assert_equal 3, character.hp
     assert_equal 10, current_skills["speed"]
   end
+
+  test "heal wounds does its work" do
+    character = create(:character)
+
+    5.times do |i|
+      character.items.create(name: "item#{i}", label: "label#{i}")
+    end
+
+    5.times do |i|
+      character.incidents.create(name: "incident#{i}", label: "label#{i+1}")
+    end
+
+    expected = [["item1", "incident0"],
+              ["item2", "incident1"],
+              ["item3", "incident2"],
+              ["item4", "incident3"]]
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Use `%w` or `%W` for an array of words.